types/react-sortable-tree/index.d.ts

```diff
@@ -20,8 +21,8 @@ export * from './utils/tree-data-utils';
 export * from './utils/default-handlers';
 
 export interface TreeItem {
-    title?: string;
-    subtitle?: string;
+    title?: React.ReactNode;
```

Comment:
Why not string?

Reply:
Title and subtitle can contain anything what can be rendered. (e.g. elements)